ssc/cmod_etes_electric_resistance.cpp

```diff
@@ -698,18 +701,27 @@ class cm_etes_electric_resistance : public compute_module
 
                     tou_params->mc_pricing.mv_is_diurnal = true;
 
-                    bool are_all_assigned = is_assigned("dispatch_sched_weekday") || is_assigned("dispatch_sched_weekend")
-                        || is_assigned("dispatch_factor1") || is_assigned("dispatch_factor2") || is_assigned("dispatch_factor3")
-                        || is_assigned("dispatch_factor4") || is_assigned("dispatch_factor5") || is_assigned("dispatch_factor6")
-                        || is_assigned("dispatch_factor7") || is_assigned("dispatch_factor8") || is_assigned("dispatch_factor9");
+                    // isn't this any assigned? Shouldn't all be and &&?
```

Comment:
Agree that this should have been && based on other context. (but please delete commented out code)